packages/theme/src/cli/utilities/notifier.ts

```diff
@@ -0,0 +1,66 @@
+import {outputDebug, outputWarn} from '@shopify/cli-kit/node/output'
+import fs from 'fs/promises'
+
+// A valid NotifyPath can be:
+// - a URL
+// - a relative or absolute path on the filesystem
+export class Notifier {
```

Comment:
I felt that errors from the notifier should not be fatal to the entire process - open to discussion